packages/infra/lib/hl7-notification-stack/vpn.ts

```
@@ -63,15 +103,78 @@ export class VpnStack extends cdk.NestedStack {
     );
 
     if (props.vpnConfig.staticRoutesOnly) {
```

I assume that by default nothing is allowed? Should we throw if this is not defined?

---

Yeah I should just delete this at this point. I was trying to understand how BGP works, but I think we'll just ship with static routing for now.